.github/workflows/check-links.yml

```diff
@@ -1,5 +1,9 @@
 name: Check links
-on: push
+
+# TODO: switch back to the push trigger if/when this workflow gets fixed; see https://github.com/ASFHyP3/hyp3-docs/issues/340
+# on: push
+on:
```

Comment:
is it supposed to be `on: workflow_dispatch` or as written? I haven't done this before it just looks different

Reply:
I don't remember the exact reason for the syntax, but I copied it from `hyp3-flood-monitoring` where it has been working.

Reply:
Ah, it's because you can specify inputs to workflow dispatch if you want: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/